tools/sqf_linter.py

```diff
@@ -0,0 +1,66 @@
+﻿#!/usr/bin/env python3
+
+# Requires: https://github.com/LordGolias/sqf
+
+import fnmatch
+import os
+import sys
+import argparse
+from sqf.parser import parse
+import sqf.analyzer
+from sqf.exceptions import SQFParserError
+
+
+def analyze(filename, writer=sys.stdout):
+    warnings = 0
+    errors = 0
+    with open(filename, 'r') as file:
+        code = file.read()
+        try:
+            result = parse(code)
+        except SQFParserError as e:
+            print("{}:".format(filename))
+            writer.write('    [%d,%d]:%s\n' % (e.position[0], e.position[1] - 1, e.message))
+            return 0, -1
```

Comment:
What's the reason this returns a `-1` for number of errors? I feel like it should be 0, but perhaps I'm missing something